### PR TITLE
Update en_US.lang

### PR DESCRIPTION
**Pull request by lex1975**
Added some localization. One bug fixed

### CODE REVIEW

**Comment by KorDum** [comment]
https://github.com/ForestryMC/Binnie/pull/94#discussion_r120001280

File: src/main/resources/assets/extratrees/lang/en_US.lang [hunk 1]
```diff
@@ -58,8 +58,11 @@ extratrees.block.planks.olive=Olive
 extratrees.block.planks.sweetgum=Sweetgum
 extratrees.block.planks.rosewood=Rosewood
 extratrees.block.planks.gingko=Gingko
-extratrees.block.planks.pinkivory=Pink Ivory
-
+# extratrees.block.planks.pinkivory=Pink Ivory
```
Comment (KorDum) [inline src/main/resources/assets/extratrees/lang/en_US.lang]: https://github.com/ForestryMC/Binnie/pull/94#discussion_r120001280

**Review by KorDum — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/extratrees/lang/en_US.lang (hunk 1 above)
Comment (lex1975) [reply src/main/resources/assets/extratrees/lang/en_US.lang]: It is right key (With a space):
extratrees.block.planks.pink ivory=Pink Ivory
I believe that this line is not needed:
extratrees.block.planks.pinkivory=Pink Ivory
Maybe I'm wrong.

Comment (KorDum) [reply src/main/resources/assets/extratrees/lang/en_US.lang]: Please remove this change and I merge PR.

File: src/main/resources/assets/extratrees/lang/en_US.lang [hunk 2]
```diff
@@ -58,8 +58,10 @@ extratrees.block.planks.olive=Olive
 extratrees.block.planks.sweetgum=Sweetgum
 extratrees.block.planks.rosewood=Rosewood
 extratrees.block.planks.gingko=Gingko
-extratrees.block.planks.pinkivory=Pink Ivory
-
+extratrees.block.planks.pink ivory=Pink Ivory
```
Comment (KorDum) [inline src/main/resources/assets/extratrees/lang/en_US.lang]: This is wrong key. Need without space.
See screenshot below.
https://cloud.githubusercontent.com/assets/1331559/26763298/558b19f2-4959-11e7-9ef2-6889bc20705c.png

**Review by KorDum — CHANGES_REQUESTED** [review]

**Comment by KorDum** [comment]
Thanks!

**Pull request merged**